lblod/frontend-gelinkt-notuleren
# Add meetings/download/copy route with just the 'copy all' button

### Overview
Do not split up the document parts yet as this way the copy-all functionality and the presence of the page can be demoed. Makes use of the download-meeting-part component, adjusted to allow for putting on the clipboard instead of only downloading. Also includes some deprecation fixes that I added...

##### connected issues and PRs:
Part of https://binnenland.atlassian.net/browse/GN-5067

### Setup
N/A

### How to test/reproduce
Go to the export options for a meeting page. All existing (just added) functionality should work, but the 'more copy options' button should take you to the new screen. After clicking 'copy full decision' the content should be on your system clipboard.

### Challenges/uncertainties
N/A

### Checks PR readiness
